doc/getting-started/getting-started.rst

```diff
@@ -126,7 +126,6 @@ To set up the ACRN build environment on the development computer:
            e2fslibs-dev \
            pkg-config \
            libnuma-dev \
-           libcjson-dev \
```

Comment:
We can't delete installing this library in the master branch documentation or else the hypervisor build will fail.  This library, installed this way, works just fine if your development system is running Ubuntu 20.04.03.
We could add a note with the different instructions you'll need to do to get this library if you're running Ubuntu 18.04.03 (as documented in https://github.com/projectacrn/acrn-hypervisor/issues/7028).

Reply:
updated